### PR TITLE
Upgrade backend to actix-web 3.0

### PR DESCRIPTION
**Pull request by maciejhirsz**

### CODE REVIEW

File: backend/src/node.rs [hunk 1]
```diff
@@ -226,7 +226,7 @@ impl Node {
         if let Ok(stringified) = serde_json::from_str::<String>(json) {
             Some(stringified.into())
         } else {
-            Some(json.into())
+            Some(json.to_owned().into())
```
Comment (dvdplm) [inline backend/src/node.rs:229]: Some docs would be good on this method, explaining the old vs new node behaviour? How long is it reasonable to support old nodes?

**Review by dvdplm — COMMENTED** [review]
Looks sane to me, but I have never used Actix so best leave the proper review to the experts.

File: backend/src/main.rs [hunk 2]
```diff
@@ -77,44 +75,59 @@ fn feed_route(
...
+        }
+    }
 }
 
-fn health(
-    aggregator: web::Data<Addr<Aggregator>>,
-) -> impl Future<Item = HttpResponse, Error = Error> {
-    aggregator.send(GetHealth).from_err().and_then(|count| {
-        let body = format!("Connected chains: {}", count);
+/// Entry check for health check monitoring bots
```
[the remark this answers is not in the document]
Comment (maciejhirsz) [reply backend/src/main.rs]: Typo indeed!

File: backend/src/node.rs (hunk 1 above)
Comment (maciejhirsz) [reply backend/src/node.rs:229]: New version of `bytes` crate does not have conversion from non-static slices to `Bytes`, so need to clone the JSON dump here. This only happens on the manual network state requests, which happen rarely.

**Review by mattrutherford — APPROVED** [review]
LGTM - actix-web is at 3.1.0 which includes some fixes for `NormalizePath` - you might be able to use this to simplify the routing. https://docs.rs/actix-web/3.0.2/actix_web/middleware/normalize/struct.NormalizePath.html

**Comment by maciejhirsz** [comment]
@mattrutherford thanks, I've updated the routes using the route macros now.

**Review by dvdplm — APPROVED** [review]

**Pull request merged**